src/main/java/org/codehaus/plexus/util/cli/Commandline.java

```diff
@@ -139,7 +139,7 @@
      * Create a new command line object.
      * Shell is autodetected from operating system
      *
-     * Shell usage is only desirable when generating code for remote execution.
+     * For Linux, shell usage is only desirable when generating code for remote execution.
```

Comment:
@michael-o : Good catch, you are right. Do you want I update the PR (there is 6 mentioned in file) ? (I'm not sure to understand, you have closed the PR).